<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialReferenceTool</class>
 <widget class="QDialog" name="SpatialReferenceTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spatial Reference System</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_filter">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clearFilter">
       <property name="text">
        <string>Clea filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add favorite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clearFavorite">
       <property name="text">
        <string>Clear favorite</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Coordinate...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget_XYCoord">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>400</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>EPSG</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Projected Coordinate Systems</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Geographic Coordinate Systems</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Favorites</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Current coordinate system:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_curCoord">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>420</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_filter</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>treeWidget_XYCoord</tabstop>
  <tabstop>plainTextEdit_curCoord</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_XYCoord</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>onItemClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>onAddFavorite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_filter</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>onFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clearFilter</sender>
   <signal>clicked()</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>onClearFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clearFavorite</sender>
   <signal>clicked()</signal>
   <receiver>SpatialReferenceTool</receiver>
   <slot>onClearFavorite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onItemClicked(QModelIndex)</slot>
  <slot>onAddFavorite()</slot>
  <slot>onFilter()</slot>
  <slot>onClearFilter()</slot>
  <slot>onClearFavorite()</slot>
 </slots>
</ui>
